### DISCUSSION TITLE
migrate milvus data to HDF5

### DISCUSSION THREAD

**Discussion by Jerry-yz** [milvus-io/milvus#32816] [Announcement]
i follow this [guide](https://milvus.io/docs/m2h.md)
but, i do not know, how to set source_milvus_path
my milvus cluster is installed k8s

**Comment by yhmo** [comment]
This tool is only for Milvus v1.x, we forgot to remove this tool from the document. In the tool description there is a note: "Currently, MilvusDM is only supported in Milvus 1.x version."
It doesn't work for Milvus v2.x.

**Reply by Jerry-yz** [reply]
ok, thanks
